runner/runners/invoke.go

```diff
@@ -131,17 +134,16 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 
 	switch st.State {
 	case execer.COMPLETE:
-		srcToDest := map[string]string{
-			stdout.AsFile(): "STDOUT",
-			stderr.AsFile(): "STDERR",
+		tmp, err := temp.NewTempDir("", "invoke")
```

Comment:
side comment: we want the temp directory to always resolve to os.TempDir() - should we remove the directory param from temp.NewTempDir()?